core/src/main/java/com/microsoft/applicationinsights/internal/profiler/GcReportingLevel.java

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.applicationinsights.internal.profiler;
+
+public enum GcReportingLevel {
+    NONE,
+    TENURED_ONLY,
+    ALL
+}
```

Comment:
this doesn't feel too intuitive from configuration perspective, I wonder if what you had (two separate booleans) was better, or if different naming would help here...